Runtime/AvatarCreator/Scripts/WebRequests/AvatarAPIRequests.cs

```diff
@@ -169,6 +173,7 @@ public async Task<AvatarProperties> CreateNewAvatar(AvatarProperties avatarPrope
 
         public async Task<byte[]> GetAvatar(string avatarId, bool isPreview = false, string parameters = null)
         {
+            ValidateAvatarId(avatarId);
```

Comment:
Did you test this out? The exception doesn't throw when you run the Avatar Creator elements scene (and manually edit the script to set avatar id to empty)

Reply:
Hmm, works for me 🤔